app/views/listings/_form.html.erb

```diff
@@ -37,59 +38,71 @@
+            <p class="crayons-field__description">Posting on behalf of an organization spends the organization's credits.</p>
+          </div>
         <% end %>
+
+        <div class="crayons-field crayons-field--checkbox">
+          <input
+            type="checkbox"
+            class="crayons-checkbox"
+            id="contact_via_connect_placeholder"
+            checked=true />
+
+          <label class="crayons-field__label" for="contact_via_connect_placeholder">
+            Connect messaging
+            <p class="crayons-field__description">Allow Users to message me via Connect.</p>
+          </label>
+        </div>
       </div>
     </div>
-      <% unless @organizations.present? %>
-        <%= javascript_packs_with_chunks_tag "listingForm", defer: true %>
-      <% end %>
-    <div class="field">
-      <%= form.label "location", "Location (If applicable for events, etc.)" %>
-      <%= form.text_field "location", placeholder: "32 characters max, plain text" %>
+
+    <% unless @organizations.present? %>
+      <%= javascript_packs_with_chunks_tag "listingForm", defer: true %>
+    <% end %>
+
+    <div class="crayons-field">
+      <%= form.label "location", "Location (If applicable for events, etc.)", class: "crayons-field__label" %>
+      <%= form.text_field "location", placeholder: "New York", class: "crayons-textfield m:max-w-50" %>
+      <p class="crayons-field__description">32 characters max, plain text</p>
     </div>
+  </div>
 
-    <div class="actions">
-      <br />
-      <p>You will not be charged credits to save a draft.</p>
-      <%= form.button "SAVE DRAFT", type: "submit", name: "listing[action]", class: "cta cta-main-listing-form cta-draft", value: "draft" %>
+  <div class="px-4 m:px-0">
+    <% unless @credits.size > 0 || (@organizations.present? && @organizations.sum(:unspent_credits_count) > 0) %>
+      <div class="crayons-notice mb-4">You need at least one credit to create a listing.</div>
+    <% end %>
+    <div class="flex mb-4 items-center">
       <% if @credits.size > 0 || (@organizations.present? && @organizations.sum(:unspent_credits_count) > 0) %>
-        <%= form.submit "PUBLISH LISTING", class: "cta cta-main-listing-form" %>
+        <%= form.submit "Publish listing", class: "crayons-btn mr-2" %>
       <% else %>
-        <p>You need at least one credit to create a listing.</p>
-        <a href="/credits/new" class="cta cta-main-listing-form" data-no-instant>Purchase Credits Now</a>
-      <% end %>
-    </div>
-    <p><em><%= community_name %> admins may modify tags and categories to ensure accuracy. We may also take down and refund inappropriate listings.</em></p>
-    <p><em>Listings expire 30 days after they are published or "bumped".</em></p>
-    <div class="listings-current-credits">
-      <% if @credits.size == 0 && (@organizations.blank? || @organizations.sum(:unspent_credits_count).zero?) %>
-        <h3>Listings Require Credits</h3>
```

Comment:
The failing spec seems to be caused by a check for this String to appear visible. Maybe `You need at least one credit to create a listing` would be the best replacement for this check 🙂